soccer/radio/Radio.hpp

```diff
@@ -12,9 +14,9 @@
  * @details This is the abstract superclass for USBRadio and SimRadio, which do
  * the actual work - this just declares the interface.
  */
-class Radio {
+class Radio : public Node {
```

Comment:
I think the best way to handle this is to leave the original `Radio` class as the interface for the two radios we have and then have a second class named RadioNode or something that wraps the `_radio` and deals with all the state info. It'll clean up the inheritance and split up any driver specific code away from the node itself.